packages/components/src/components/modal/component.tsx

```diff
@@ -109,14 +109,14 @@ export class KolModal implements Generic.Element.ComponentApi<RequiredProps, Opt
 	@Prop() public _ariaLabel!: string;
 
 	/**
-	 * Setzt die Breite des Modals. (max-width: 100%).
+	 * Gibt die EventCallback-Function für das Schließen des Modals an.
 	 */
-	@Prop() public _width?: string = '100%';
+	@Prop() public _on?: KoliBriModalEventCallbacks;
 
 	/**
-	 * Gibt die EventCallback-Function für das Schließen des Modals an.
+	 * Setzt die Breite des Modals. (max-width: 100%).
 	 */
-	@Prop() public _on?: KoliBriModalEventCallbacks;
+	@Prop() public _width?: string = '100%';
```

Comment:
Das scheint nicht hier reingehören. Ggf. Modal PR

Reply:
Die sind nur in der Reihenfolge verschoben.